src/components/themed/TransactionListRow.tsx

```diff
@@ -150,21 +150,23 @@ export function TransactionListRow(props: Props) {
 
   // Pending Text and Style
   const currentConfirmations = transaction.confirmations
-  const isConfirmed = currentConfirmations === 'confirmed'
 
-  const unconfirmedOrTimeText = isConfirmed
-    ? unixToLocaleDateTime(transaction.date).time
-    : !isSentTransaction && canReplaceByFee && currentConfirmations === 'unconfirmed'
-    ? lstrings.fragment_transaction_list_unconfirmed_rbf
-    : currentConfirmations === 'unconfirmed'
-    ? lstrings.fragment_wallet_unconfirmed
-    : currentConfirmations === 'dropped'
-    ? lstrings.fragment_transaction_list_tx_dropped
-    : typeof currentConfirmations === 'number'
-    ? sprintf(lstrings.fragment_transaction_list_confirmation_progress, currentConfirmations, requiredConfirmations)
-    : lstrings.fragment_transaction_list_tx_synchronizing
+  const unconfirmedOrTimeText =
+    currentConfirmations === 'confirmed'
+      ? unixToLocaleDateTime(transaction.date).time
+      : !isSentTransaction && canReplaceByFee && currentConfirmations === 'unconfirmed'
+      ? lstrings.fragment_transaction_list_unconfirmed_rbf
+      : currentConfirmations === 'unconfirmed'
+      ? lstrings.fragment_wallet_unconfirmed
+      : currentConfirmations === 'dropped'
+      ? lstrings.fragment_transaction_list_tx_dropped
+      : currentConfirmations === 'failed'
+      ? lstrings.fragment_transaction_list_tx_failed
```

Comment:
I realize the code was like this already, but maybe we should take this huge ? : block and turn it into a utility function, rather than keep shoving one more thing in to an already over-stuffed situation. That way we can use a simpler `if` / `return` style:

```ts
function getConfirmationText(tx: EdgeTransaction) {
  if (tx.confirmations === 'confirmed') {
    return unixToLocaleDateTime(tx.date).time
  }

  if (tx.confirmations === 'unconfirmed') {
    if (!isSentTransaction && canReplaceByFee) {
      return lstrings.fragment_transaction_list_unconfirmed_rbf
    }
    return lstrings.fragment_wallet_unconfirmed
  }

  if (tx.confirmations === 'failed') {
  ...
}
```